DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/mmhost/sample.json

```diff
@@ -0,0 +1,71 @@
+{
+  "guid": "CE509751-B84A-44D1-A391-2007937F3A7F",
+  "name": "MMHost",
+  "categories": ["Toolkit/oneAPI Direct Programming/C++SYCL FPGA/Tutorials/Features/experimental"],
+  "description": "An Intel® FPGA tutorial demonstrating the usage of mmhost macros",
+  "toolchain": ["icpx"],
+  "os": ["linux", "windows"],
+  "targetDevice": ["FPGA"],
+  "builder": ["ide", "cmake"],
+  "languages": [{"cpp":{}}],
+  "commonFolder": {
+    "base": "../../../..",
+    "include": [
+      "README.md",
+      "Tutorials/Features/experimental/mmhost",
```

Comment:
@ajaykumarkannan are we planning to keep mm host properties as 'experimental' in 2024.0? I support this, but we should be clear with the steering committee etc. if that is the case.

Reply:
Yes, all properties will stay in experimental for 2024.0.